prime-select.sh

```diff
@@ -299,8 +313,9 @@ case $type in
         fi
         
         if (( service_test == 0)); then
-	    
-	    if ! [ -f /etc/systemd/system/multi-user.target.wants/prime-select.service ]; then
+
+        if ! [ -f /etc/systemd/system/multi-user.target.wants/prime-select.service ]; then
```

Comment:
Why this change?

Reply:
This hunk shouldn't be there.

Reply:
This hunk shouldn't be there.

Reply:
I don't like such a change in the script.

Reply:
It wasn't suppose to be. I thought this hunk was removed from my commit yesterday evening. I will rebase and put it back the way it was. It wasn't intentional!